src/test/java/flab/gotable/controller/MemberAPIControllerTest.java

```diff
@@ -0,0 +1,89 @@
+package flab.gotable.controller;
+
+import flab.gotable.domain.entity.Member;
+import flab.gotable.dto.ApiResponse;
+import flab.gotable.dto.request.MemberSignUpRequestDto;
+import flab.gotable.exception.DuplicatedIdException;
+import flab.gotable.mapper.MemberMapper;
+import flab.gotable.service.MemberService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+import java.util.HashMap;
+import java.util.Map;
+
+class MemberAPIControllerTest {
+
+    private MemberAPIController memberAPIController;
+
+    @BeforeEach
+    void setup() {
+        Map<String, Member> memberMap = new HashMap<>();
+
+        memberAPIController = new MemberAPIController(
+                new MemberService(new MemberMapper() {
```

Comment:
memberMap 으로 유사 DB를 구현하셨네요~ 이 map이 `MemberService` 내부에 있는게 더 좋지 않을까요?
```suggestion
        

        memberAPIController = new MemberAPIController(
          new MemberService(new MemberMapper() {
            final Map<String, Member> memberMap = new HashMap<>();
```